### ISSUE TITLE
Controls icons load but never appear

### ISSUE THREAD

**Issue by kolpax** [clappr/clappr#898]
**Browser**: Chrome Version 49.0.2623.87 (64-bit)

**OS**: OSX 10.11.3 (15D21)

**Clappr Version**: 0.2.38

When loading a page with clappr I do not see the controls icons. I can still click on their spaces and play/pause/other but the SVG icons do not appear. I can see in the Network dev tools that `sprite.svg` loads correctly.
I am using clappr inside of a CycleJS component, the source looks kind of like this:

```javascript
const div = document.createElement('div')
div.classList.add('video')
this.player = new clappr.Player({
  parent: div,
  width: '100%',
  height: '100%',
  hlsMinimumDvrSize: 5,
})
this.player.core.$el[0].style.position = 'absolute'
this.player.load(this.state.source)
this.player.play()
```

I could not reproduce this issue at http://cdn.clappr.io/ which makes me think it is something with my server or client configuration.

<img width="480" alt="screen shot 2016-03-16 at 17 08 31" src="https://cloud.githubusercontent.com/assets/3372392/13819453/bebd1bce-eb99-11e5-9ae2-3db71eb8ce57.png">

<img width="898" alt="screen shot 2016-03-16 at 17 10 10" src="https://cloud.githubusercontent.com/assets/3372392/13819503/01f3aac0-eb9a-11e5-86ba-854ffec6b8b0.png">

**Comment by bikegriffith** [comment]
I've struggled with this with my React app as well.  2 things come to mind:
1. Check your console to make sure there are no cross-origin errors for the font face
2. You may want to hack around and make sure the Player icon font is added to your <head> manually ... when you tear down the DOM nodes for the player, it seemed to confuse it

```
// By default, Clappr injects the font into a style tag inside the MediaControl
// and Poster plugins, but these elements getting created/destroyed can cause the
// typeface to disappear.  We'll just load it into the <head> and be done.


import { MediaControl } from 'clappr';

export function ensurePlayerFontLoaded(baseUrl) {
  try {
    var media = new MediaControl({
      baseUrl: baseUrl
    });
    var style = media.stylesheet;
    appendStyleTagIfNotAlreadyInHead(style[0]);
    media.destroy();
  } catch(e) {
    console.error('Unable to load Player font:', e);
    console.error(e.stack);
  }
}

function appendStyleTagIfNotAlreadyInHead(style) {
  var c = style.className; //should be "clappr-style"
  if (document.querySelectorAll(`head .${c}`).length) {
    return;
  } else {
    document.getElementsByTagName('head')[0].appendChild(style);
  }
}
```

**Comment by towerz** [comment]
@kolpax @bikegriffith we are working in an updated version of the media control/poster that do not rely on font rendering, instead using inline svgs for the icons. I believe this will be added in the next clappr release.

**Comment by bikegriffith** [comment]
@towerz would love to know if that helps with the total minified bundle download size ... that's one of our current concerns

**Comment by towerz** [comment]
@bikegriffith it probably won't reduce the total minified bundle size, because it will add the svg source directly to the media control template, but it will reduce the total download size  (and perhaps the the inline css size), because no font will be added to either the controls or the poster. The svg files we use are really small, and they will be minified as well.

Even if it doesn't reduce the total size, there's another improvement we're working on that will shrink the bundle size. Most of the built-in plugins will be extracted from the core player, as well as the default media control, and they will become standalone plugins. This will help reduce bundle sizes for whatever project that include clappr in their bundle, and make it more flexible to customize what plugins should be used for a given player instance.

**Comment by towerz** [comment]
This has been fixed in 0.2.41, but feel free to reopen if this issue is still happening.
